<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoughParm</class>
 <widget class="QDialog" name="HoughParm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1663</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>2</y>
     <width>1646</width>
     <height>886</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QSpinBox" name="bin_thresh">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QSpinBox" name="canny_thresh1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QSpinBox" name="canny_thresh2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="singleStep">
       <number>0</number>
      </property>
      <property name="value">
       <number>120</number>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QSpinBox" name="canny_Ssize">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="gray_pic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>544</width>
        <height>408</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>gray picture</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="6">
     <widget class="QLabel" name="bin_pic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>544</width>
        <height>408</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>binary picture</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10" colspan="4">
     <widget class="QLabel" name="canny_pic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>544</width>
        <height>408</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>canny picture</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSpinBox" name="hough_thresh">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>3264</number>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="hough_minlen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>3264</number>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="hough_maxgap">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>3264</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="shi_quality">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QSpinBox" name="shi_mindist">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>3224</number>
      </property>
      <property name="value">
       <number>35</number>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QSpinBox" name="shi_Bsize">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QDoubleSpinBox" name="shi_hough_k">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.040000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2" colspan="4">
     <widget class="QLabel" name="line_pic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>544</width>
        <height>408</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>get Line</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" rowspan="2" colspan="6">
     <widget class="QLabel" name="corner_pic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>544</width>
        <height>408</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>get corners</string>
      </property>
     </widget>
    </item>
    <item row="3" column="13">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="12">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="13">
     <widget class="QPushButton" name="exit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="hough_centroid">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Show Centroid</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QCheckBox" name="shi_useHough">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QCheckBox" name="shi_centroid">
      <property name="text">
       <string>Show Centroid</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
